isolate/test/index.ts

```diff
@@ -492,11 +492,11 @@ describe('isolate', function() {
         {other: new MyTestSource()},
         `foo`,
         `bar`
-      );
+      ) as {other: Observable<Array<string>>};
```

Comment:
If Isolate were aware of Most.js and RxJS types of streams, we could support this test to type check without any changes. So we have to manually cast to RxJS's stream type here, otherwise it would be `unknown`.